docs/deploy-to-vercel.md

```diff
@@ -1,27 +1,13 @@
 # Deploy to Vercel
 
-[Vercel](https://vercel.com) (formerly ZEIT) is a cloud platform for websites and serverless APIs, that you can use to deploy your Gridsome projects to your personal domain (or a free `.now.sh` suffixed URL).
+[Vercel](https://vercel.com) is a cloud platform that enables developers to host Jamstack websites and web services that deploy instantly, scale automatically, and requires no supervision, all with zero configuration. They provide a global edge network, SSL encryption, asset compression, cache invalidation, and more.
 
-This guide will show you how to get started in a few quick steps:
+### Step 1: Deploying your Gridsome Website to Vercel
```

Comment:
Can we adjust the headings? There's a Step 1 but no step 2. Also, it skips from an h1 to h3.

Maybe it could be like:

```
# Deploy to Vercel

## Overview

Vercel is a cloud platform...

## Configuring Deployments with Vercel

To deploy your Gridsome app...
```

Reply:
Fixing!

Reply:
It still has step 1 but no step 2.

Reply:
Fixing!